Childmonitoring.txt

```diff
@@ -5,7 +5,7 @@
 LiquidCrystal lcd(12, 11, 5, 4, 3, 2);

```

Comment:
1) Naming Convention: Variable names should be descriptive and use camelCase. Consider renaming 'DHTPIN' to 'dhtPin' and 'DHTTYPE' to 'dhtType'.
2) Constants: Use 'const' keyword for constants like DHTPIN and DHTTYPE to prevent accidental modification.
3) Comments: Add comments to explain the purpose of the code, especially for the LiquidCrystal lcd initialization.
4) Spacing: Add proper spacing between operators and operands for better readability.
5) Functionality: The code snippet seems incomplete. Consider adding setup() and loop() functions to initialize the components and read the sensor values.